apis/exoscale/v1/mysql_types.go

```diff
@@ -0,0 +1,115 @@
+package v1
+
+import (
+	"reflect"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/crossplane-runtime/pkg/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// MySQLParameters are the configurable fields of a MySQL.
+type MySQLParameters struct {
+	Maintenance MaintenanceSpec `json:"maintenance,omitempty"`
+	Backup      BackupSpec      `json:"backup,omitempty"`
+
+	// +kubebuilder:validation:Enum=ch-gva-2;ch-dk-2;de-fra-1;de-muc-1;at-vie-1;bg-sof-1
+	// +kubebuilder:validation:Required
+
+	// Zone is the datacenter identifier in which the instance runs in.
+	Zone string `json:"zone"`
```

Comment:
```suggestion
	// +kubebuilder:validation:Required

	// Zone is the datacenter identifier in which the instance runs in.
	Zone Zone `json:"zone"`
```

since #50 there's a specific type for this

Reply:
weird, I updated it a few minutes ago 🤔 
as I amended to previous commit, it was cached